packages/cli/src/credentials/oauth2Credential.api.ts

```diff
@@ -162,6 +163,17 @@ oauth2CredentialController.get(
 			returnUri = `${get(oauthCredentials, 'authUrl', '') as string}?${percentEncoded}`;
 		}
 
+		if (oauthCredentials.grantType === 'pkce') {
+			let { code_verifier, code_challenge } = pkceChallenge();
+			if (code_verifier.includes('~')) {
+				do {
+					({ code_verifier, code_challenge } = pkceChallenge());
+				} while (code_verifier.includes('~'));
+			}
```

Comment:
Let's add a comment that we need this loop because the twitter api does not accept '~' character in code_verifier.